### ISSUE TITLE
DateTime::Duration::add() broken since 1.37

### ISSUE THREAD

**Issue by ppisar** [houseabsolute/DateTime.pm#50]
DateTime::Duration:add() does not work since DateTime-1.37:

```
$ perl -Ilib -MDateTime::Duration -e '$a=$b=DateTime::Duration->new(hours=>1, minutes=>0); $a->add($b)'
Expected a hash or hash reference but got a single object argument
```

**Comment by preaction** [comment]
This should be fixed by #46 which was released in DateTime 1.41. Could you try upgrading and see if that fixes your problem?

**Comment by autarch** [comment]
@preaction - I think this is a different bug. This is about calling `->add` on a `DateTime::Duration` object.

**Comment by preaction** [comment]
Sorry, my bad, that it is.

**Comment by autarch** [comment]
Fixed in 1.42.
